### ISSUE TITLE
ui bootstrap dropdown not working

### ISSUE THREAD

**Issue by kobiburnley** [ngbp/ngbp#378]
```
<!-- Split button -->
<div class="btn-group" dropdown>
  <button type="button" class="btn btn-danger">Action</button>
  <button type="button" class="btn btn-danger dropdown-toggle" dropdown-toggle>
    <span class="caret"></span>
    <span class="sr-only">Split button!</span>
  </button>
  <ul class="dropdown-menu" role="menu">
    <li><a href="#">Action</a></li>
    <li><a href="#">Another action</a></li>
    <li><a href="#">Something else here</a></li>
    <li class="divider"></li>
    <li><a href="#">Separated link</a></li>
  </ul>
</div>
```

dropdown isn't opening

**Comment by saibotsivad** [comment]
Please create a complete demo on [plunker](http://plnkr.co/) so we can see what is wrong with your code.

Since this is demo code from [here](http://angular-ui.github.io/bootstrap/#/dropdown), did you make sure to follow the [getting started instructions](http://angular-ui.github.io/bootstrap/#/getting_started)?

If their example is bad, you should probably file an issue in the [ui-boostrap repo](https://github.com/angular-ui/bootstrap/issues) instead, you'll probably get better assistance.

**Comment by kobiburnley** [comment]
the problem was angular-bootstrap dependency in bower.json file.
has to be "angular-bootstrap": "~0.12.1"

ngbp default is an earlier version, as can be seen here:
https://github.com/ngbp/ngbp/blob/v0.3.2-release/bower.json

**Comment by herrherrmann** [comment]
Yes, I also think the `angular-bootstrap` and `bootstrap` versions should be updated in the bower.json. Additionally the new angular-bootstrap will require [angular-animate](https://docs.angularjs.org/api/ngAnimate#!) to be included as well.

**Example:**

``` javascript
"devDependencies": {
  "angular":           "~1.4.3",
  "bootstrap":         "~3.3.5",
  "angular-bootstrap": "~0.13.2",
  "angular-animate":   "~1.4.3"
  // some more stuff
}
```
